index.html

```diff
@@ -2229,7 +2238,96 @@ <h3>Global State</h3>
  configured. The default value of the flag is true if the endpoint
  doesn't support proxy configuration, or false otherwise.</p>
 
-<p>When asked to <dfn>close the session</dfn>,
+<div>
+
+<p>To <dfn>create a session</dfn>, given a JSON
+Object <var>capabilites</var>, and a set <var>flags</var>:
+<ol>
+ <li><p>Let <var>session id</var> be the result of <a>generating a UUID</a>.
+
+ <li><p>Let <var>session</var> be a new <a>session</a> with <a>session
+  ID</a> <var>session id</var>, and <a>HTTP flag</a> <var>flags</var> contains
+  "<code>http</code>".
+
+ <li><p>Let <var>proxy</var> be the result of getting property
+     "<code>proxy</code>" from <var>capabilities</var> and run the
+     substeps of the first matching statement:
+
+       <dl class=switch>
+         <dt><var>proxy</var> is a <a>proxy configuration</a> object</dt>
+         <dd><p>Take implementation-defined steps to set the user agent proxy
+             using the extracted <var>proxy</var> configuration. If the
+             defined proxy cannot be configured return <a>error</a> with
+             <a>error code</a> <a>session not created</a>. Otherwise
+             set the <a>has proxy configuration</a> flag to true.</dd>
+
+         <dt>Otherwise</dt>
+         <dd><a>Set a property</a> of <var>capabilities</var> with name
+           "<code>proxy</code>" and a value that is a new JSON <a>Object</a>.</dd>
+       </dl>
+   </li>
+
+  <li><p>If <var>capabilites</var> has a property named
+  "<code>acceptInsecureCerts</code>", set the <a>remote
+  end</a>'s <a>accept insecure TLS</a> flag to the result
+  of <a>getting a property</a> named
+  "<code>acceptInsecureCerts</code>" from <var>capabilities</var>.
```

Comment:
As I read above we already configure the proxy, and here we only set the flag but don't install the handler. Shouldn't the proxy configuration happen later similar to when we set the insecure cert handler?